src/mcts/params.cc

```diff
@@ -279,6 +282,7 @@ void SearchParams::Populate(OptionsParser* options) {
   options->Add<IntOption>(kMaxCollisionEventsId, 1, 1024) = 32;
   options->Add<IntOption>(kMaxCollisionVisitsId, 1, 1000000) = 9999;
   options->Add<BoolOption>(kOutOfOrderEvalId) = true;
+  options->Add<IntOption>(kMaxOutOfOrderEvalsId, 1, 10000) = 256;
```

Comment:
Also override this value to 32 for selfplay games, here:
https://github.com/LeelaChessZero/lc0/blob/master/src/selfplay/tournament.cc#L114
So that selfplay doesn't change (but match games actually still will be changed).

Otherwise, looks good!

Reply:
I set the default to 32, but raising it could allow better GPU utilization in self play.